<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(192, 255, 207);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 25pt &quot;Yu Gothic UI Semibold&quot;;
background-color: rgb(254, 255, 169)</string>
   </property>
   <property name="text">
    <string>Удаление</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(254, 255, 169)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Yu Gothic UI Semilight&quot;;</string>
   </property>
   <property name="text">
    <string>дата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Yu Gothic UI Semilight&quot;;</string>
   </property>
   <property name="text">
    <string>сумма дохода</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Yu Gothic UI Semilight&quot;;</string>
   </property>
   <property name="text">
    <string>сумма расхода</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
background-color: rgb(248, 255, 169)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Yu Gothic UI Semilight&quot;;</string>
   </property>
   <property name="text">
    <string>статья дохода/расхода</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="doxod">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
background-color: rgb(248, 255, 169)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="rasxod">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
background-color: rgb(248, 255, 169)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="state">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
background-color: rgb(248, 255, 169)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addbtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>470</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
background-color: rgb(85, 255, 0);
border-radius: 20px;
color: white;</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>dateEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>doxod</zorder>
  <zorder>rasxod</zorder>
  <zorder>state</zorder>
  <zorder>addbtn</zorder>
  <zorder>error</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
